battleship.py

```diff
@@ -25,7 +25,16 @@
 Returns: None
 '''
 def makeModel(data):
-    return
+    data["no.of rows"]=10

```

Comment:
variable names should be changed, it should be _ instead of dots. pls change.

Reply:
changed mam